blocks/search/search.css

```diff
@@ -0,0 +1,80 @@
+.search .search-box {
+  display: flex;
+  align-items: center;
+}
+
+.search .search-box input {
+  width: calc(100% - 30px);
```

Comment:
if we use `grid` (or even `flex`), we can omit this width calculation.
```suggestion
```

Reply:
Done. Thank you! 👍